File: 백준/Bronze/3009. 네 번째 점/네 번째 점.js
const input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n").map(el => el.split(" ").map(Number));

let x = input.map((v) => v[0]).sort((a, b) => a - b);
let y = input.map((v) => v[1]).sort((a, b) => a - b);

x = x[0] === x[1] ? x[2] : x[0];
y = y[0] === y[1] ? y[2] : y[0];

console.log(x, y);

File: 백준/Bronze/3009. 네 번째 점/네 번째 점.py
dots = []
x = []
y = []

for _ in range(3):
    dots.append(list(map(int, input().split())))

for [a, b] in dots:
    if a in x:
        x.remove(a)
    else:
        x.append(a)
    
    if b in y:
        y.remove(b)
    else:
        y.append(b)

print(x[0], y[0])